### ISSUE TITLE
Delete task with backspace 

### ISSUE THREAD

**Issue by thstyl2000** [quillpad/quillpad#281]
As one can add a new task by enter, one should also be able to delete an empty task with backspace.

Thank you for for this great app!

**Comment by TanguyP** [comment]
I agree, and I would add that if you place the caret at the beginning of a non-empty task, backspace should merge it with the preceding task if that exists.

For example, if you have this (^ sign indicates the location of the caret):
- This task was accidentally split 
- ^intwo lines

Then if you press backspace, it should result in this:
- This task was accidentally split ^in two lines

That would seem logical to me at least.

**Comment by CurbedLarry** [comment]
Was going to raise this as a bug.  If you do enter on a task name it always creates a new task but you don't always want one.  It's then impossible to delete that unnamed task (without completing and deleting all completed tasks).

<img width="648" height="566" alt="Image" src="https://github.com/user-attachments/assets/98501190-c402-4e21-8bc4-bb22dc1c78f1" />

Alternative implementation would be un-named tasks auto-delete when you tap away from it.

**Comment by CurbedLarry** [comment]
Have just seen via a pull request that tasks are deleted by swiping left or right.  This isn't very intuitive.

That pull request would disable it and add a delete icon: https://github.com/quillpad/quillpad/pull/575
